clitools.php

```diff
@@ -13,6 +13,38 @@
 $log->pushHandler(new StreamHandler('php://stdout', Logger::DEBUG));
 $GLOBALS['LOG_HANDLERS'][] = new StreamHandler('php://stdout', Logger::DEBUG);
 
+function reminders_list() {
+    global $log;
+    list($slack_credentials, $caldav_credentials, $agenda_args) = read_config_file();
+    $api = new SlackAPI($slack_credentials['bot_token'], $slack_credentials['user_token']);
+    $user_infos = $api->auth_test("user");
+    foreach($api->reminders_list() as $id => $reminder) {
+        if($user_infos->user_id === $reminder["creator"]) { // delete only reminders that have been created by the app
```

Comment:
It looks like a copy/paste typo: the comment should say "list" instead of "delete"

Reply:
Of course: efc6b4b